src/Store/StreamDoctrineDbalStore.php

```diff
@@ -366,10 +392,13 @@ public function configureSchema(Schema $schema, Connection $connection): void
             ->setNotnull(true);
         $table->addColumn('playhead', Types::INTEGER)
             ->setNotnull(false);
-        $table->addColumn('event', Types::STRING)
+        $table->addColumn('event_id', Types::STRING)
+            ->setLength(255)
+            ->setNotnull(true);
+        $table->addColumn('event_name', Types::STRING)
             ->setLength(255)
             ->setNotnull(true);
```

Comment:
Did you test if the migration will be generated correctly? Renaming `event` to `event_name` and adding a new column `event_id`?

Reply:
Working renaming in migration is not relevant because auto migration will not work because event_id is required.
But that doesn't matter because the store is defined as experimental.

Reply:
I know it's experimental, but i still think we should make this upgrade as smooth as possible if we are introducing BC's or updating the tables.

Regarding the ID, yes i know the user needs to add these themselfs. We still should guide the user how he could approach this, since a classical way like this would not work. 😄 

```php
$allMessages = $store->load();

forearch ($allMessages as $message) {
    $message->withHeader(new EventIdHeader(Uuid::uuid7()->toString()));
}

$store->save($allMessages);
```

Reply:
I will write something about it in the release notes